CHANGELOG.md

```diff
@@ -79,6 +79,7 @@ Here is an overview of all new **experimental** features:
 - **General**: Fix CVE-2023-39325 in golang.org/x/net ([#5122](https://github.com/kedacore/keda/issues/5122))
 - **General**: Fix otelgrpc DoS vulnerability ([#5208](https://github.com/kedacore/keda/issues/5208))
 - **General**: Prevented stuck status due to timeouts during scalers generation ([#5083](https://github.com/kedacore/keda/issues/5083))
+- **General**: Update CloudEvent content to reflect correct emitter and subject ([#5278](https://github.com/kedacore/keda/issues/5278))
```

Comment:
This feature is not shipped yet, so no need to list here

Reply:
I've included the issue in the original entry